notebooks/data-sources/TestGrid/metrics/unexpected_test_failures.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>Change the last line to better explain the example. "So if a column has 2 tests that failed, and the first had a row score of 2/5...."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/448/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Could you please describe the intuition behind the approach in a bit more detail? E.g. what does sum of entropies signify? why are we looking at entropies of row scores for failed tests?</p>